underfs/oss/README.md

```diff
@@ -0,0 +1,63 @@
+# Configuring Tachyon with OSS
+
+This guide describes how to configure Tachyon with [Aliyun OSS](http://www.aliyun.com/product/oss/?spm=5176.383338.201.8.yfBFvm&lang=en) as the under storage system. Object Storage Service (OSS) is a massive, secure and highly reliable cloud storage service provided by Aliyun.
+
+## Initial Setup
+
+To run a Tachyon cluster on a set of machines, you must deploy Tachyon binaries to each of these machines.You can either [compile the binaries from Tachyon source code](http://tachyon-project.org/documentation/master/Building-Tachyon-Master-Branch.html), or [download the precompiled binaries directly](http://tachyon-project.org/documentation/master/Running-Tachyon-Locally.html).
+
+Then if you haven't already done so, create your configuration file from the template:
+
+    cp conf/tachyon-env.sh.template conf/tachyon-env.sh
+
+Also, in preparation for using OSS with tachyon, create a bucket or use an existing bucket. You should also note that the directory you want to use in that bucket, either by creating a new directory in the bucket, or using an existing one. For the purposes of this guide, the OSS bucket name is called OSS_BUCKET, and the directory in that bucket is called OSS_DIRECTORY. Also, for using the OSS Service, you should provide an oss endpoint to specify which range your bucket is on. The endpoint here is called OSS_ENDPOINT, and to learn more about the endpoints for special range you can see [here](http://intl.aliyun.com/docs#/pub/oss_en_us/product-documentation/domain-region). For more information about OSS Bucket, Please see [here](http://intl.aliyun.com/docs#/pub/oss_en_us/product-documentation/function&bucket)
+
+## Confugurating Tachyon
+
+To configure Tachyon to use OSS as its under storage system, modifications to the ``` conf/tachyon-env.sh``` file must be made. The first modification is to specify an existing OSS bucket and directory as the under storage system. You can specify it by modifying ```conf/tachyon-env.sh``` to include:
```

Comment:
can you change <pre>\`\`\` conf/tachyon-env.sh\`\`\`</pre> to be <pre>\`conf/tachyon-env.sh\`</pre> (i.e., replacing triple quotes to single quote and removes the leading space before conf)  and also apply the quote change to  conf/tachyon-env.sh? The reason is single quote in github markdown keeps the quoted text inline which is consistent with other markdown pages in our site.